app/services/grant_services/new.rb

```diff
@@ -17,9 +17,9 @@ def self.call(grant:, user:)
     rescue ActiveRecord::RecordInvalid => invalid
       OpenStruct.new(success?: false,
                      messages: invalid.record.errors.full_messages)
-    rescue ServiceError => err # throws an active record error
+    rescue ServiceError => invalid
```

Comment:
ServiceError needs to be changed to the new class name

Reply:
Thanks. Updated in 1282f7b below